src/Sign.Core/Resources.resx

```diff
@@ -190,8 +190,8 @@
   <data name="FetchingCertificate" xml:space="preserve">
     <value>Fetching certificate from Azure Key Vault.</value>
   </data>
-  <data name="InvalidSha1ThumbrpintValue" xml:space="preserve">
-    <value>Invalid value for {0}. The value must be the SHA-1 thumbprint of a certificate in a Windows Certificate Store.</value>
+  <data name="InvalidCertificateFingerprintValue" xml:space="preserve">
+    <value>Invalid value for {0}. The value must be a {1} fingerprint of a certificate in a Windows Certificate Store.</value>
```

Comment:
You updated this resource string to have 2 string replacements instead of 1, but the call site still passes 1 argument.